config/core/resources/containersource.yaml

```diff
@@ -31,13 +31,8644 @@ spec:
+                                      on a resource or set of resources. Clients must treat
+                                      these values as opaque and passed unmodified back to the
+                                      server. They may only be valid for a particular resource
+                                      or set of resources.  Populated by the system. Read-only.
+                                      Value must be treated as opaque by clients and . More
+                                      info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency'
+                                  type: string
+                              selfLink:
+                                  description: 'SelfLink is a URL representing this object. Populated
+                                      by the system. Read-only.  DEPRECATED Kubernetes will
+                                      stop propagating this field in 1.20 release and the field
+                                      is planned to be removed in 1.21 release.'
+                                  type: string
+                              spec:
+                                  description: 'Specification of the desired behavior of the
+                                      pod. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status'
+                                  type: object
+                                  properties:
+                                      activeDeadlineSeconds:
+                                          description: 'Optional duration in seconds the pod
+                                              may be active on the node relative to StartTime
+                                              before the system will actively try to mark it
+                                              failed and kill associated containers. Value must
+                                              be a positive integer.'
+                                          type: integer
+                                          format: int64
+                                      affinity:
+                                          description: 'If specified, the pod''s scheduling constraints'
+                                          type: object
+                                          properties:
+                                              nodeAffinity:
+                                                  description: 'Describes node affinity scheduling
+                                                      rules for the pod.'
+                                                  type: object
+                                                  properties:
+                                                      preferredDuringSchedulingIgnoredDuringExecution:
+                                                          description: 'The scheduler will prefer
```

Comment:
This block is repeated identically 3 times, you can use placeholders to avoid duplication as I did in https://github.com/knative/eventing/pull/3869 (see https://github.com/knative/eventing/pull/3869/files#diff-360926796c72432f2956878c2251f3d8R221)

Reply:
On further analysis, looks like that block isn't duplicated entirely the same. There are a few words that have been changed between blocks - like "affinity" vs "non-affinity"....so unfortunately, I can't use a placeholder in this example. Looking for more dup blocks....